mysql/resource_ti_config_variable.go

```diff
@@ -0,0 +1,171 @@
+package mysql
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/creasty/defaults"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/tidwall/gjson"
+)
+
+func resourceTiConfigVariable() *schema.Resource {
+	return &schema.Resource{
+		Create: CreateOrUpdateConfigVariable,
+		Read:   ReadConfigVariable,
+		Update: CreateOrUpdateConfigVariable,
+		Delete: DeleteConfigVariable,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"value": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
```

Comment:
Does changing a value need `ForceNew` and therefore destruction of the resource?

Reply:
True. I'll fix it!